mailpoet/lib/EmailEditor/Integrations/MailPoet/EmailEditor.php

```diff
@@ -65,4 +77,14 @@ public function extendEmailPostApi() {
       'schema' => $this->emailApiController->getEmailDataSchema(),
     ]);
   }
+
+  public function replaceEditor($replace, $post) {
+    $currentScreen = get_current_screen();
+    if (use_block_editor_for_post($post) && $post->post_type === self::MAILPOET_EMAIL_POST_TYPE && $currentScreen) { // phpcs:ignore Squiz.NamingConventions.ValidVariableName.MemberNotCamelCaps
```

Comment:
[Nitpick]
Since we are already checking for the MailPoet custom post type. Do we still need to use the `use_block_editor_for_post`?  
On checking the [docs](https://developer.wordpress.org/reference/functions/use_block_editor_for_post/), the function has a filter that can be used to disable the block editor for certain (or all post type), is that a behavior we expect and want?

Reply:
> On checking the [docs](https://developer.wordpress.org/reference/functions/use_block_editor_for_post/), the function has a filter that can be used to disable the block editor for certain (or all post type), is that a behavior we expect and want?

Nice catch! We want to always use block editor so the `use_block_editor_for_post` makes no sense. Removed in https://github.com/mailpoet/mailpoet/pull/5930/commits/475ef6d9df01f18c046d6a5cbab8bc4d7560e696